spec/unit/tariff_synchronizer/taric_update_spec.rb

```diff
@@ -183,31 +179,20 @@
     it { is_expected.to eq('TGB21203.xml') }
   end
 
-  describe '.applicable_updates' do
-    subject(:applicable_updates) { described_class.applicable_updates.as_json }
+  describe '.applicable_update' do
+    subject(:applicable_update) { described_class.applicable_update.as_json }
```

Comment:
does not it break when  `.applicable_update` returns nil?

Reply:
Niet